### PR TITLE
Cmake folder exception exception

### LINKED ISSUE #886: prepareGaudiExec needs warning for pre-existing folder
Reminder to add a warning for prepareGaudiExec in case a checked out version is already there.

### PR DESCRIPTION
**Pull request by mesmith75**
Fixes #886 . For the prepare_cmake function.
Check whether the Dev folder already exists. If it does, raise a GangaException rather than continuing with the job submission.

### CODE REVIEW

File: python/GangaLHCb/Lib/LHCbDataset/LHCbDataset.py [hunk 1]
```diff
@@ -339,8 +339,8 @@ def optionsString(self, file=None):
                 sdatasetsnew += """ \"LFN:%s\",""" % f.lfn
                 sdatasetsold += """ \"DATAFILE='LFN:%s' %s\",""" % (f.lfn, dtype_str)
             else:
-                sdatasetsnew += """ \"PFN:%s\",""" % f.namePattern
-                sdatasetsold += """ \"DATAFILE='PFN:%s' %s\",""" % (f.namePattern, dtype_str)
+                sdatasetsnew += """ \"%s\",""" % f.accessURL()[0]
```
Comment (rob-c) [inline python/GangaLHCb/Lib/LHCbDataset/LHCbDataset.py]: I'm not complaining about adding it in, but it seems unrelated to the issue unless I'm missing something?

**Comment by mesmith75** [comment]
Any objections to merging?
